main.js

```diff
@@ -0,0 +1,71 @@
+// object constructor
+function BooksConstructor(
+  title,
+  author,
+) {
+  this.title = title;
+  this.author = author;
+}
+// all books will be saved into the next array;
+const books = [
+
+];
+
+// displaying books
+
+function display() {
+  const tbody = document.querySelector('.booklist');
+  tbody.innerHTML = '';
+  let index = 0;
+  books.forEach((element) => {
+    const currentRow = document.createElement('tr');
+    const currentTitle = element.title;
+    const currentAuthor = element.author;
+    currentRow.innerHTML = `${currentTitle}<br>${currentAuthor} <button class='removeButton' value="${index}">Remove</button> <hr>`;
+    tbody.appendChild(currentRow);
+    index += 1;
+  });
+
+  // remove book from the books
+  function removeBook(event) {
+    const indexToRemove = event.target.value;
+    books.splice(indexToRemove, 1);
+    display();
+  }
+  // display after removeing book
+
+  const removebtn = document.querySelectorAll('.removeButton');
+  removebtn.forEach((element) => {
+    element.addEventListener('click', removeBook);
+  });
+}
+
+// function to add books;
+function addBooks() {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  books.push(new BooksConstructor(title, author));
+  display();
+}
+
+const addBookButton = document.querySelector('#addBook');
+addBookButton.addEventListener('click', addBooks);
+
+// presarving data to the local storage
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+bookTitle.addEventListener('keydown', () => {
+  localStorage.setItem('title', bookTitle.value);
+});
+bookAuthor.addEventListener('keydown', () => {
+  localStorage.setItem('Author', bookAuthor.value);
+});
+const browserTitle = localStorage.getItem('title');
+const browserAuthor = localStorage.getItem('Author');
+if (browserTitle) {
+  bookTitle.value = browserTitle;
+}
+
+if (browserAuthor) {
+  bookAuthor.value = browserAuthor;
+}
```

Comment:
- [ ] Good job on storing the value of the inputs field using local storage, but, there is no need for that. Kindly store the collection of books in the local storage instead as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Make%20sure%20that%20data%20is%20preserved%20in%20the%20browser%27s%20memory%20by%20using%20localStorage.), to reload it after the page is refreshed.